src/main/java/jenkins/plugins/office365connector/TimeUtils.java

```diff
@@ -0,0 +1,69 @@
+package jenkins.plugins.office365connector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang.time.FastDateFormat;
+
+/**
+ * Collects method for time and duration.
+ *
+ * @author Damian Szczepanik (damianszczepanik@github)
+ */
+public final class TimeUtils {
+
+    /**
+     * Formatter for date. Uses local timezone and locale.
+     */
+    private static final FastDateFormat DATE_FORMAT = FastDateFormat.getInstance("yyyy-MM-dd HH:mm:ss z");
+
+    private TimeUtils() {
+    }
+
+    /**
+     * Converts data to formatted string.
+     *
+     * @param date date to format
+     * @return formatted date
+     */
+    public static String dateToString(long date) {
+        return DATE_FORMAT.format(date);
+    }
+
+    /**
+     * Converts duration to formatted string.
+     * @param duration duration to convert
+     * @return formatted duration
+     */
+    public static String durationToString(long duration) {
+        long diff[] = new long[4];
+        List<String> formats = new ArrayList<>();
+
+        // sec
+        diff[3] = (duration >= 60 ? duration % 60 : duration);
+        if (diff[3] > 0) {
```

Comment:
... except this ``if``s that skip if value is zero - check junit test below

This is probably the only change of functionality in this PR